shell.nix

```diff
@@ -1,5 +1,6 @@
-{ nixpkgs ? import ./.nix/nixpkgs.nix
+{ nixpkgs ? <nixpkgs-unstable> # import ./.nix/nixpkgs.nix
```

Comment:
Is this right?

Reply:
... maybe 

![homer](https://i.giphy.com/media/a93jwI0wkWTQs/200.webp)